book/tutorials/nb-to-package/intro.md

```diff
@@ -0,0 +1,270 @@
+# Packaging notebook introduction and demo
+
+This tutorial demonstrates how powerful packaging your Python code
+can be ... even for yourself.
+
+Before we get ahead of ourselves. Let's think about the **WHY**.
+You might ask yourself:
+
+- why would I want to package my code?
+- can't I just pass around notebooks?
+- what's the point of all this?
+- what is the meaning of life?
+
+... well, maybe not the last one. But the others are valid questions.
+
+## What is a Python package?
+
+A "package" or officially called a "distribution package" is simply
+a directory of Python modules and sub-packages with an `__init__.py` file
+that allows for the modular organization of code.
+It can be be installed and distributed for use by others.
```

Comment:
```suggestion
A "package" is simply an organized directory of Python files that can be be installed and distributed for use by others.
```

Reply:
(I recommend not including "module" and "sub-package" in the definition until those are defined). An alternative might be to flip the order of presentation to start with a function and build up to package